hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestKafkaSource.java

```diff
@@ -191,13 +191,13 @@ public void testJsonKafkaSourceWithDefaultUpperCap() {
      */
     testUtils.sendMessages(TEST_TOPIC_NAME, Helpers.jsonifyRecords(dataGenerator.generateInserts("000", 1000)));
     InputBatch<JavaRDD<GenericRecord>> fetch1 = kafkaSource.fetchNewDataInAvroFormat(Option.empty(), Long.MAX_VALUE);
-    assertEquals(500, fetch1.getBatch().get().count());
+    assertEquals(1000, fetch1.getBatch().get().count());
```

Comment:
why exactly does this test have to change? could you please clarify

Reply:
> why exactly does this test have to change? could you please clarify
hi @vinothchandar,  the value of `maxEventsToReadFromKafka `has changed in this commit, so the value of `fetch1.getBatch().get().count() `also changes in the test case.

> there is an empty `git` file in this commit.. can you please remove this?
@vinothchandar I hive removed this file.

Reply:
Hi @Trevor-zhang Would please add a new test case to verify the `sourceLimit` less than the generated insert record's num?

Reply:
> Hi @Trevor-zhang Would please add a new test case to verify the `sourceLimit` less than the generated insert record's num?

@yanghua  ok,i'd like to.